### PR TITLE
Partfix 4092

### PR DESCRIPTION
**Pull request by Charlotte-Morgan**
### What does it fix ?
* Ticket : #4092
* Funded by : DFAT
* Description : 
Add icons for age, gender and vulnerability concepts and update license file:
infant, child, child_bearing, pregnant_lactating, under_5, over_60, disabled, displaced

### CODE REVIEW

**Comment by Charlotte-Morgan** [comment]
@ismailsunni - the test and fix pep 8 should not be in this PR

**Comment by ismailsunni** [comment]
@Charlotte-Morgan no worries. As long as the whole PR only contains the important changes, we can always squash the PR.

**Comment by felix-yew** [comment]
Hi @Charlotte-Morgan - I try this PR and I just can check from the infographic result to see the icon, the result like this:
image
So, I can not see the update icon.

**Comment by Charlotte-Morgan** [comment]
@felix-yew - what do you think the problem is?
I have not changed any min needs stuff only population, age, gender and vulnerability

**Comment by felix-yew** [comment]
The icons of population: age, gender and vulnerability. Only female that I can see 😄 on that infographic result.

**Comment by felix-yew** [comment]
> can you check your report with dev checkout to be sure its this PR please

@Charlotte-Morgan - I check with this PR and also with develop. And, it produce the same result as the image that I share above for the population analysis icon.

**Comment by Charlotte-Morgan** [comment]
@felix-yew  - thanks for checking. If I understand your bug is not related to this PR then?

**Comment by Charlotte-Morgan** [comment]
hi @ismailsunni - could you please review the tests that will break with changes to default values.
So far there is one based on default female ratio
```
Traceback (most recent call last):
  File "/tests_directory/safe/impact_function/test/test_impact_function.py", line 488, in test_ratios_with_raster_exposure
    displaced_population * female_ratio, female_displaced)
AssertionError: 813.4399999999999 != 813
```

**Comment by Charlotte-Morgan** [comment]
@samnawi - can you please look at the changes in default_values.py to check the values I have used?

**Comment by Charlotte-Morgan** [comment]
@timlinux - please jump in, especially if this is holding things up.

**Comment by timlinux** [comment]
```
======================================================================
FAIL: test_ratios_with_raster_exposure (impact_function.test.test_impact_function.TestImpactFunction)
Test if we can add defaults to a raster exposure.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/tests_directory/safe/impact_function/test/test_impact_function.py", line 488, in test_ratios_with_raster_exposure
    displaced_population * female_ratio, female_displaced)
AssertionError: 813.4399999999999 != 813
----------------------------------------------------------------------
```

The issue is in here:

```
        # Let's check computation
        index = analysis.fieldNameIndex(
            displaced_field['field_name'])
        displaced_population = analysis.uniqueValues(index)[0]
        self.assertEqual(
            displaced_population * female_ratio, female_displaced)
```

I think we can probably just cast those to ints in the last line so:

```
      int(displaced_population * female_ratio), female_displaced)
```

**Comment by Charlotte-Morgan** [comment]
yay :)

File: safe/definitions/default_values.py [hunk 1]
```diff
@@ -13,23 +13,27 @@
...
     # total population: 1.01 male(s)/female (2011 est.)
-    'default_value': 0.5,
+    'default_value': 0.496,
+    # Updated for InaSAFE 4.1
+    # UNSD World Data, 2010.
     'min_value': 0,
     'max_value': 1,
     'increment': 0.001,
-    'description': tr('Default value for female ratio')
+    'description': tr('Default ratio of females per 100 people in the '
```
Comment (ismailsunni) [inline safe/definitions/default_values.py]: Pull left

File: safe/definitions/default_values.py [hunk 2]
```diff
@@ -66,106 +70,175 @@
...
     'name': tr('Infant Ratio Global Default'),
-    'default_value': 0.0,  # FIXME:It's random number
+    'default_value': 0.0925,
+    # http://data.un.org/Data.aspx?d=PopDiv&f=variableID%3a30
+    # UNSD World Data, 2010.
     'min_value': 0,
     'max_value': 1,
     'increment': 0.001,
-    'description': tr('Default value for infant ratio')
+    'description': tr('Default ratio of infants per 100 people in the '
```
Comment (ismailsunni) [inline safe/definitions/default_values.py]: Pull left

File: safe/definitions/default_values.py [hunk 3]
```diff
@@ -66,106 +70,175 @@
...
     'name': tr('Child Ratio Global Default'),
-    'default_value': 0.0,  # FIXME:It's random number
+    'default_value': 0.1735,
+    # http://data.un.org/Data.aspx?d=PopDiv&f=variableID%3a31
+    # UNSD World Data, 2010.
     'min_value': 0,
     'max_value': 1,
     'increment': 0.001,
-    'description': tr('Default value for child ratio')
+    'description': tr('Default ratio of children per 100 people in the '
```
Comment (ismailsunni) [inline safe/definitions/default_values.py]: Pull left

File: safe/definitions/default_values.py [hunk 4]
```diff
@@ -66,106 +70,175 @@
...
-    'default_value': 0.254,
+    # Updated for InaSAFE 4.1
+    # http://data.un.org/Data.aspx?d=PopDiv&f=variableID%3a101
+    # UNSD World Data, 2010.
+    'default_value': 0.266,
     'min_value': 0,
     'max_value': 1,
     'increment': 0.001,
-    'description': tr('Default value for youth ratio')
+    'description': tr('Default ratio of youths per 100 people in the '
```
Comment (ismailsunni) [inline safe/definitions/default_values.py]: Pull left

File: safe/definitions/default_values.py [hunk 5]
```diff
@@ -66,106 +70,175 @@
...
-    'default_value': 0.659,
+    # Updated for InaSAFE 4.1
+    # http://data.un.org/Data.aspx?d=PopDiv&f=variableID%3a103
+    # UNSD World Data, 2010.
+    'default_value': 0.679,
     'min_value': 0,
     'max_value': 1,
     'increment': 0.001,
-    'description': tr('Default value for adult ratio')
+    'description': tr('Default ratio of adults per 100 people in the '
```
Comment (ismailsunni) [inline safe/definitions/default_values.py]: Pull left

File: safe/definitions/default_values.py [hunk 6]
```diff
@@ -66,106 +70,175 @@
...
-    'default_value': 0.087,
+    # Updated for InaSAFE 4.1
+    # http://data.un.org/Data.aspx?d=PopDiv&f=variableID%3a103
+    # UNSD World Data, 2010.
+    'default_value': 0.077,
     'min_value': 0,
     'max_value': 1,
     'increment': 0.001,
-    'description': tr('Default value for elderly ratio')
+    'description': tr('Default ratio of elderly people per 100 people in the '
```
Comment (ismailsunni) [inline safe/definitions/default_values.py]: Pull left

File: safe/definitions/default_values.py [hunk 7]
```diff
@@ -66,106 +70,175 @@
...
-    'default_value': 0.0,  # FIXME:It's random number
+    'default_value': 0.093,
+    # Updated for InaSAFE 4.1
+    # http://data.un.org/Data.aspx?d=PopDiv&f=variableID%3a30
+    # UNSD World Data, 2010.
     'min_value': 0,
     'max_value': 1,
     'increment': 0.001,
-    'description': tr('Default value for under 5 years old ratio')
+    'description': tr('Default ratio of under 5 year olds per 100 people in '
```
Comment (ismailsunni) [inline safe/definitions/default_values.py]: Pull left

**Pull request merged**